src/libsyntax/attr.rs

```diff
@@ -1291,7 +1291,7 @@ impl HasAttrs for StmtKind {
     fn attrs(&self) -> &[Attribute] {
         match *self {
             StmtKind::Local(ref local) => local.attrs(),
-            StmtKind::Item(..) => &[],
+            StmtKind::Item(ref item) => item.attrs(),
```

Comment:
What is this "fixing"? It may be the case that this change will result in the same attributes being seen *twice*.
Although it's in libsyntax. Where it might not even be used directly.